lib/Drawing/Color.vala

```diff
@@ -24,31 +24,31 @@ using Granite.Services;
 namespace Granite.Drawing {
 
     /**
-     * A class containing an RGBA color and methods for more powerful color manipulation.
+     * A class containing an rgba color and methods for more powerful color manipulation.
      */
     public class Color : GLib.Object, SettingsSerializable {
 
         /**
          * The value of the red channel, with 0 being the lowest value and 1.0 being the greatest value.
          */
-        public double R;
+        public double r;
```

Comment:
I'm unsure whether or not this will cause breakage elsewhere. Is there a good way to test this change out to ensure I didn't break anything in the process? Everything builds safely when running `ninja -C build`

Reply:
Since this is a library, this implies an API break for apps that are using Granite. Probably better to leave it where it is

Reply:
:+1: i'll revert the name-convention changes